discovery_engine_core/core/src/storage.rs

```diff
@@ -286,4 +301,11 @@ pub mod models {
         pub(crate) size: u32,
         pub(crate) next_page: u32,
     }
+
+    #[derive(Debug, PartialEq)]
+    pub(crate) struct TimeSpendDocumentView {
```

Comment:
:wrench: past time, as in the engine
```suggestion
    pub(crate) struct TimeSpentDocumentView {
```